html/pfappserver/lib/pfappserver/Form/Config/Source/Stripe.pm

```diff
@@ -0,0 +1,79 @@
+package pfappserver::Form::Config::Source::Stripe;
+
+=head1 NAME
+
+pfappserver::Form::Authentication::Source::Stripe
+
+=cut
+
+=head1 DESCRIPTION
+
+pfappserver::Form::Authentication::Source::Stripe
+
+=cut
+
+use strict;
+use warnings;
+use HTML::FormHandler::Moose;
+use pf::Authentication::Source::StripeSource;
+use pf::log;
+extends 'pfappserver::Form::Config::Source::Billing';
+with 'pfappserver::Base::Form::Role::Help';
+
+# Form fields
+has_field 'secret_key' => (
+    type => 'Text'
+);
+
+has_field 'publishable_key' => (
+    type => 'Text'
+);
+
+has_field 'style' => (
+    type    => 'Select',
+    default => 'charge',
+    options => [{label => 'Charge', value => 'charge'}, {label => 'Subscription', value => 'subscription'}],
+    tags => { after_element => \&help,
+              help => 'The type of payment the user will make. Charge is a one time fee, subscription will be arecurring fee.' },
```

Comment:
He said its not a word, but he actually cheated because he googled it

He said it should either be 'subscription will be a recurring fee' or 'subscription must be paid in form of turkey sandwiches'. He said you would know which one to pick